honeybee_vtk/writer.py

```diff
@@ -0,0 +1,141 @@
+"""Write vtk files to the disk."""
+
+import os
+import json
+import warnings
+
+from zipfile import ZipFile
+from honeybee.typing import clean_string
+from .hbjson import read_hbjson
+from .hbjson import check_grid, get_grid_base, get_grid_mesh, get_grid_points
+from .tovtk import group_by_face_type
+from .tovtk import write_polydata, write_color_grouped_points, write_arrows
+from .helper import get_face_center
+
+
+def write_vtk(
+        file_path, *, file_name=None, get_grids=True,
+        get_vectors=True):
+
+    """Reads a valid HBJSON and writes a .zip of vtk files to the disk.
+
+    Args:
+        file_path: A text string for a valid path to the HBJSON file.
+        file_name: A text string for the name of the .zip file to be written. If no
+            text string is provided, the name of the HBJSON file will be used as a
+            file name for the .zip file.
+        get_grids: A boolean. Defaults to True. Grids will not be extracted from HBJSON
+            if set to False.
+        get_vectors: A boolean. Defaults to True. Vector arrows will not be created if
+            set to False.
+    """
+    # HBJSON file
+    if file_path:
+        # Check if path is fine
+        
+        if not os.path.isfile(file_path):
+            raise FileNotFoundError(
+                'The path is not a valid path.'
+                ' If file exists, try using double backslashes in file path'
+                ' and try again.'
+            )
+    try:
+        # Check if the file is a valid JSON
+        with open(file_path) as fp:
+            hbjson = json.load(fp)
+    except json.decoder.JSONDecodeError:
+        raise ValueError(
+            'Not a valid JSON file.'
+            )
+    else:
```

Comment:
so many if/else clauses here. makes it hard to read the code. I think you can just continue the code here without using else.